ipapython/ipautil.py

```diff
@@ -282,6 +282,37 @@ def write_tmp_file(txt):
 
     return fd
 
+
+@contextmanager
+def concurrent_open(path, mode='r'):
```

Comment:
build-in open uses 'name' in python 2 and 'file' in python 3. So why not? I can combine it to 'filename'.